export-tiles/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
```

Comment:
appears to be unused, should be removed.

Reply:
removed